### PR TITLE
For issue #10231 update proton icons third batch

### PR DESCRIPTION
**Pull request by Amejia481**
As there are multiple icons getting updated at the same time, I provided @brampitoyo a custom APK for checking all of them in detail. AC related changes https://github.com/mozilla-mobile/android-components/pull/10249

- The pencil icon is still pending as it's not yet on Figma.
- The rating is been a little tricky to replace, I'm still working on it.

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/drawable/ic_location_disabled.xml [hunk 1]
```diff
@@ -1,27 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_location_disabled.xml:1]: Undo this

File: app/src/main/res/drawable/ic_location_enabled.xml [hunk 2]
```diff
@@ -1,27 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_location_enabled.xml:1]: Undo

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml [hunk 3]
```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<animated-vector
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:drawable="@drawable/ic_notifications_enabled">
-    <target android:name="strike_thru_path" android:animation="@animator/strike_thru_path_disable" />
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_notifications_anim_disable.xml:8]: Can we also remove `strike_thru_path_disable.xml`

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml [hunk 4]
```diff
@@ -1,11 +0,0 @@
...
-    <target android:name="strike_thru_mask" android:animation="@animator/strike_thru_mask_disable" />
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_notifications_anim_disable.xml:9]: Same as above.

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml [hunk 5]
```diff
@@ -1,11 +0,0 @@
...
-    <target android:name="icon" android:animation="@animator/fill_disable" />
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_notifications_anim_disable.xml:10]: Same as above.

File: app/src/main/res/drawable/ic_notifications_anim_enable.xml [hunk 6]
```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<animated-vector
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:drawable="@drawable/ic_notifications_disabled">
-    <target android:name="strike_thru_path" android:animation="@animator/strike_thru_path_enable" />
-    <target android:name="strike_thru_mask" android:animation="@animator/strike_thru_mask_enable" />
-    <target android:name="icon" android:animation="@animator/fill_enable" />
```
Comment (gabrielluong) [inline app/src/main/res/drawable/ic_notifications_anim_enable.xml:10]: Same as aboove.

**Review by gabrielluong — APPROVED** [review]

File: app/src/main/res/drawable/ic_notifications_anim_enable.xml (hunk 6 above)
Comment (Amejia481) [reply app/src/main/res/drawable/ic_notifications_anim_enable.xml:10]: Removed. Thanks!

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml (hunk 5 above)
Comment (Amejia481) [reply app/src/main/res/drawable/ic_notifications_anim_disable.xml:10]: Removed. Thanks!

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml (hunk 4 above)
Comment (Amejia481) [reply app/src/main/res/drawable/ic_notifications_anim_disable.xml:9]: Removed. Thanks!

File: app/src/main/res/drawable/ic_notifications_anim_disable.xml (hunk 3 above)
Comment (Amejia481) [reply app/src/main/res/drawable/ic_notifications_anim_disable.xml:8]: Removed. Thanks!

**Review by brampitoyo — COMMENTED** [review]
r+

I’ve also sent the link to the newly delivered `edit` icon to @Amejia481.

**Pull request merged**